odk1-src/aggregate-vm.rst

```diff
@@ -1,4 +1,65 @@
-ODK Aggregate Virtual Machine 
-=================================
+.. spelling::
 
-The `ODK Aggregate VM <https://gumroad.com/l/odk-aggregate-vm>`_ is a fully-configured install of Aggregate that you can run on any computer. It requires very little setup, works well without Internet connectivity, and gives you complete control over your data collection campaign.
+  readme
+  VirtualBox
+
+ODK Aggregate Virtual Machine
+=============================
+
+This document provides instructions on setting up ODK Aggregate using the `ODK Aggregate VM OVA file <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ and `VirtualBox <https://www.virtualbox.org>`_.
+
+.. admonition:: Before you get started…
+
+  - Please read this entire document before installing the virtual machine (VM). Doing so will save you a lot of time and effort.
+
+Basic Usage
+-----------
+
+1. Download and install VirtualBox from `virtualbox.org <https://www.virtualbox.org>`_.
+2. Double-click `ODK-Aggregate-v1.5.0-beta.0.ova <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ to import the VM. Accept the default settings.
+3. After the import completes, start the VM.
+4. After the VM starts, wait for Welcome to ODK Aggregate VM message to be shown.
+5. Do not login into the black and white screen of the VM. Instead on your computer, open a web browser.
+6. Go to the web address shown in the VM boot screen (e.g., `http://localhost:10080 <http://localhost:10080>`_). You will see the Aggregate login screen.
+7. Click "Sign in with Aggregate password" and login with username and password shown below.
+
+  - Aggregate username: aggregate
+  - Aggregate password: aggregate
+
+8. ODK Aggregate will remind you to change your administrator password. Please do so!
+
+.. admonition:: Notes
+
+  - If after unzipping the installer and launching the VM, a "File is busy" or "File access error" message is reported on Windows, it is likely due to improper unzipping or write permissions. Try right-clicking the zip file, then select Extract All and save the files to a folder on the Desktop.
+
+  - If on Linux you get an error message about "PAE: Unable to boot", make sure VirtualBox has PAE/NX enabled. That setting is usually under System/Processor.
+
+  - The ova file is the installer. Once the VM has imported, the ova file and the zip file can be deleted. It is a good idea to keep the readme (this file).
+
+Securing the VM
+---------------
+
+Once the VM reboots, login to the command line interface with the following credentials
+
+- Username: root
+- Password: aggregate
+
+You will be forced to change the default password on login. Please choose a secure password!
+
+Connecting an external device (e.g., ODK Collect) to the VM
```

Comment:
Avoid Latin abbreviations. This could probably just be **Connecting to Collect**

(Also needs a section label.)